lib/verdict/experiment.rb

```diff
@@ -52,6 +52,17 @@ def groups(segmenter_class = Verdict::Segmenters::FixedPercentageSegmenter, &blo
     return self
   end
 
+  # Optional: Together with the "end timestamp", limits the experiment run timeline within
+  # the given time interval. When experiment is not scheduled, subject switch returns nil.
+  # This is useful when the experimenter requires the experiment to run in a strict timeline.
+  def schedule_start_timestamp(timestamp)
+    @schedule_start_timestamp = timestamp
```

Comment:
are we making any assumptions about timezone here? is everything gonna be in UTC?

Reply:
Good point. No assumptions are being made at this level because we are dealing with Ruby Date objects which encapsulate timezone information.